src/functions_framework/__init__.py

```diff
@@ -215,6 +225,7 @@ def create_app(target=None, source=None, signature_type=None):
     # 5. Create the application
     app = flask.Flask(target, template_folder=template_folder)
     app.config["MAX_CONTENT_LENGTH"] = MAX_CONTENT_LENGTH
+    app.register_error_handler(500, crash_handler)
```

Comment:
Should this be all 500-level status codes or just `500`?

Reply:
This is primarily for uncaught exceptions, which produce a 'crash' message in GCF Python 3.7. From my understanding, all of these produce 500 response codes in Flask?

Reply:
Yep